### PR TITLE
fix: members count trigger when creating a squad

### PR DESCRIPTION
**Pull request by sshanzel**
There is an instance where the user does not start being a member, that is when creating a new Squad. To make it more accurate, we check whether the user is not blocked.

### CODE REVIEW

File: src/migration/1726147441577-SourceMembersCountFix.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        RETURN NEW;
+      END;
+      $$
+    `);
+    queryRunner.query(
+      `
+      CREATE OR REPLACE TRIGGER increment_squad_members_count
+      AFTER INSERT ON "source_member"
+      FOR EACH ROW
+      WHEN (NEW.role != 'blocked')
```
Comment (sshanzel) [inline src/migration/1726147441577-SourceMembersCountFix.ts:31]: The only difference from the previous one is this condition.

https://github.com/dailydotdev/daily-api/blob/main/src/migration/1725981526888-SourceFlagMembersIncrement.ts

**Review by nensidosari — APPROVED** [review]

**Comment by sshanzel** [comment]
@capJavert let me know what you think 👀

File: src/migration/1726147441577-SourceMembersCountFix.ts (hunk 1 above)
Comment (capJavert) [reply src/migration/1726147441577-SourceMembersCountFix.ts:31]: down should have the old function and trigger definition :hidethepain: .. its so hard to update triggers in migrations

**Review by capJavert — APPROVED** [review]
All good, just one comment to make a valid migration.

File: src/migration/1726147441577-SourceMembersCountFix.ts (hunk 1 above)
Comment (sshanzel) [reply src/migration/1726147441577-SourceMembersCountFix.ts:31]: Ah, right. Yep. Definitely needs to created on the down function 😅

**Pull request merged**